### PR TITLE
Make tmp db file absolute

### PR DESCRIPTION
**Pull request by bonomat**
It seems that relative path-based execution leads to errors from time to time where the db file can't be found. Using an absolute path solves this problem. The script needs to be executed from within sqlite-db module anyway to produce the sqlx-data.json file. Hence, I think this is a justified solution.

This ticket is related: https://github.com/launchbadge/sqlx/issues/1399

### CODE REVIEW

**Review by da-kami — COMMENTED** [review]
Is it expected that the `sqlite-db/sqlx-data.json` file changes when only changing the path to the database? I would have not expected that 😅

**Comment by bonomat** [comment]
> Is it expected that the `sqlite-db/sqlx-data.json` file changes when only changing the path to the database? I would have not expected that 😅

No, it means someone else did not check it in before me.

**Review by luckysori — COMMENTED** [review]
Could the second commit be because of a different version of `sqlx`?

**Comment by bonomat** [comment]
> Could the second commit be because of a different version of `sqlx`?

Not sure, but I'm preparing a ci target to ensure that the file is always up2date: It's currently being tested here: https://github.com/bonomat/hermes/runs/7444408411?check_suite_focus=true

File: sqlite-db/prepare_db.sh [hunk 1]
```diff
@@ -15,4 +14,4 @@ trap 'rm -f $TEMPDB' EXIT
 DATABASE_URL=sqlite:$TEMPDB cargo sqlx migrate run
 
 # prepare the sqlx-data.json rust mappings
-DATABASE_URL=sqlite:./$CRATE_DIR/$TEMPDB SQLX_OFFLINE=true cargo sqlx prepare -- --tests
+DATABASE_URL=sqlite:$TEMPDB SQLX_OFFLINE=true cargo sqlx prepare -- --tests
```
Comment (klochowicz) [inline sqlite-db/prepare_db.sh]: classic bash nit 🙃

```suggestion
DATABASE_URL=sqlite:${TEMPDB} SQLX_OFFLINE=true cargo sqlx prepare -- --tests
```

**Review by klochowicz — APPROVED** [review]
LGTM!

**Comment by bonomat** [comment]
CI check works 😅 
https://github.com/itchysats/itchysats/runs/7460197517?check_suite_focus=true

**Review by luckysori — APPROVED** [review]

**Comment by bonomat** [comment]
> Build failed:
> 
>     * [formatting](https://github.com/itchysats/itchysats/runs/7460558400?check_suite_focus=true)

? Works locally and the other ci job was also fine?

**Comment by bonomat** [comment]
bors retry

**Pull request merged**